pkg/ovs/ovsctl/ofctl.go

```diff
@@ -18,21 +18,30 @@ import (
 	"bufio"
 	"bytes"
 	"fmt"
+	"regexp"
 	"strconv"
 	"strings"
 )
 
+func (c *ovsCtlClient) DumpTables() ([]string, error) {
+	tableDump, err := c.RunOfctlCmd(false, "dump-tables")
```

Comment:
could you add a comment about why we do not set the version to Openflow13 for this specific command?

Reply:
Added.

Reply:
I think you mean "dump-tables" and not "dump-flows"
Do you know why that is? Was that intentional from the OVS developers?

Reply:
Good catch. I don't know the reason now, and I didn't find related documents after searching. I'll make an issue to OVS team about this.

Reply:
Thanks for the explanation from @igsilya

> OpenFlow 1.3 breaks down the OFPST_TABLE request in two separate requests, OFPMT_TABLE only reports table statistics, and OFPMT_TABLE_FEATURES reports features, names, and some other things. See the A.3.5.4 Table Statistics section in OF1.2 and OF1.3 specs and the A.3.5.5 Table Features in the OF1.3 spec.
So, since ovs-ofctl just directly implements OF commands, in order to get table names with OpenFlow 1.3 you need to use ovs-ofctl dump-table-features.